src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/admin/staff/courtworker/CourtWorkerService.java

```diff
@@ -40,25 +41,25 @@ public List<String> addEmployeeMember(AdminData adminData) {
 
     private boolean checkIfEmployeeMemberExists(CourtWorker courtWorker, TribunalOffice tribunalOffice,
                                                 List<String> errors) {
-        if (courtWorkerRepository.existsByTribunalOfficeAndTypeAndCode(tribunalOffice, CourtWorkerType.EMPLOYEE_MEMBER,
+        if (courtWorkerRepository.existsByTribunalOfficeAndTypeAndCode(tribunalOffice, courtWorker.getType(),
```

Comment:
Would it be a good idea to get tribunalOffice by courtWorker.getTribunalOffice()?

Reply:
refactored